doc/refcard/refcard.tex

```diff
@@ -288,18 +294,57 @@
   \columnbreak
 %% ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  \Sect{ESS developer}
+  \Sect{Package development}
 %% ~~~~~~~~~~~~~~~~
-  Evaluate your code into the package (in \textbf{\underline{ess-dev-map}}:
-  \textbf{C-c C-t}):
-
+  Evaluate your code into a given package (in \texttt{ess-dev-map},
+  \texttt{C-c C-t}):
+  
+  \begin{tabI}
+    \texttt{C-s, s} & Select package for namespaced evaluation\\
+  \end{tabI}
+  
+  Keybindings in \texttt{ess-r-package-mode} (\texttt{C-c C-w})\\
+  (call to \texttt{devtools} functions):
+  
   \begin{tabI}
-    \texttt{C-t, t} & Toggle developer on/off\\
-    \texttt{C-a, a} & Add package to the dev list\\
-    \texttt{C-r, r} & Remove package from dev list\\
+    \texttt{C-b, b} & Build package (\texttt{devtools::build()})\\
+    \texttt{C-c C-c} & Check package (\texttt{devtools::check()})\\
+    \texttt{C-d, d} & Document package (\texttt{devtools::document()})\\
+    \texttt{C-i, i} & Install package (\texttt{devtools::install()})\\
+    \texttt{C-t, t} & Run all tests (\texttt{devtools::test()})\\
   \end{tabI}
 
 
+%% All these commands seems to be obsolete:
+%%  \begin{tabI}
+%%    \texttt{C-t, t} & Toggle developer on/off\\
+%%    \texttt{C-a, a} & Add package to the dev list\\
+%%    \texttt{C-r, r} & Remove package from dev list\\
+%%  \end{tabI}
+  
+  \Sect{Editing documentation}
+  
+  Inside \texttt{*.Rd} files:
+  
+  \begin{tabI}
+    \texttt{C-c C-e} & Insert a template of help file\\
+    \texttt{\TAB} & Indent current line\\
+    \texttt{C-c C-f} & Display list of font specifiers\\
+    \texttt{C-c C-j} & Insert an `\texttt{\textbackslash item}' on next line\\
+    \texttt{C-c C-n} & (in example section) \\
+                     & send current line to ESS process \\
+    \texttt{C-c C-p} & Preview help file in plain text \\
+  \end{tabI}
+
+  For \texttt{Roxygen} users, inside \texttt{*.R} files (\textbf{C-c C-o}):
```

Comment:
shall we make it `(prefix C-c C-o)`. Also elsewhere?

Reply:
Thanks, done in this section and elsewhere.